### PR TITLE
Make lazyjson lazier to deal with memory usage.

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

File: conda_forge_tick/utils.py [hunk 1]
```diff
@@ -130,63 +131,66 @@ def __getitem__(self, name: Any) -> str:
...
 
     def __setitem__(self, key: Any, value: Any) -> None:
-        self._load()
-        assert self.data is not None
         self.data[key] = value
         self._dump()
 
     def __getstate__(self) -> dict:
         state = self.__dict__.copy()
-        state["data"] = None
```
Comment (beckermr) [inline conda_forge_tick/utils.py:189]: This is for pickling IIRC. Do we need to clear the class level cache in the state?

Comment (mariusvniekerk) [reply conda_forge_tick/utils.py:189]: the class state isn't the instance state, so should be fine

Comment (beckermr) [reply conda_forge_tick/utils.py:189]: great

**Comment by CJ-Wright** [comment]
@mariusvniekerk the tests are unhappy

**Pull request merged**